src/spikeinterface/sorters/external/mountainsort5.py

```diff
@@ -114,8 +111,13 @@ def _setup_recording(cls, recording, sorter_output_folder, params, verbose):
     @classmethod
     def _run_from_folder(cls, sorter_output_folder, params, verbose):
         import mountainsort5 as ms5
+        from mountainsort5.util import TemporaryDirectory, create_cached_recording
 
         recording = cls.load_recording_from_folder(sorter_output_folder.parent, with_warnings=False)
+        if recording is None:
+            raise Exception("Unable to load recording from folder.")
+        if not isinstance(recording, BaseRecording):
+            raise Exception("Unexpected: recording extracted from folder is not a BaseRecording")
```

Comment:
```suggestion
            raise TypeError(f"Unexpected: recording extracted from folder is not a BaseRecording, but is of type: {type(recording)}")
```

Reply:
I think this will make debugging in the future easier if some complains and they can tell us what is being read instead of the `BaseRecording`.